macros/unpack/get_exposures.sql

```diff
@@ -11,21 +11,22 @@
 
         {%- for node in nodes_list -%}
 
-          {%- set values_line %}
-
-            '{{ node.unique_id }}',
-            '{{ node.name }}',
-            '{{ node.resource_type }}',
-            '{{ node.original_file_path }}',
-            cast('{{ dbt_project_evaluator.is_not_empty_string(node.description) | trim }}'as boolean),
-            '{{ node.type }}',
-            '{{ node.maturity}}',
-            '{{ node.package_name }}',
-            '{{ node.url }}',
-            '{{ node.owner.name }}',
-            '{{ node.owner.email }}'
-
-          {% endset -%}
+          {%- set values_line = 
+            [
+              "'" ~ node.unique_id ~ "'",
```

Comment:
is there any way for us to make this a little less repetitive? i recently learned about `adapter.quote` -- not sure it even works here but could be cool

Reply:
I think that `adapter.quote` is targeted at quoting database object names (e.g. surrounding the db name with `` for BQ, etc...), which is different from how to quote strings.

I get your point though and will create a macro for this use case to make it clearer than using the complex pattern everywhere.

Reply:
I have pushed a commit with a new macro